biorbd_optim/constraints.py

```diff
@@ -127,8 +127,28 @@ def __proportional_variable(ocp, nlp, V, policy):
 
     @staticmethod
     def __contact_force_inequality(ocp, nlp, X, U, policy):
-        for i in range(len(U)):
-            contact_forces = Dynamics.get_forces_from_contact(X[i], U[i], nlp)
+        """
+        A compléter, notamment policy est soit un tuple ou un tuple de tuples, avec en 1er indice
+        le numéro de la force de contact et en 2e indice la borne associée
```

Comment:
Even though this is for yourself, write your comment in english. So if you ever forget, this does not lie there forever ;)

Reply:
Done in commit 12959992ef966abe6ceb5969ea1e50577220a7f2